sysdig/resource_sysdig_secure_custom_policy.go

```diff
@@ -123,14 +123,66 @@ func customPolicyToResourceData(policy *v2.Policy, d *schema.ResourceData) {
 		_ = d.Set("type", "falco")
 	}
 
-	rules := []map[string]interface{}{}
+	rules := getPolicyRulesFromResourceData(d)
+	newRules := []map[string]interface{}{}
 	for _, rule := range policy.Rules {
-		rules = append(rules, map[string]interface{}{
+		newRules = append(newRules, map[string]interface{}{
 			"name":    rule.Name,
 			"enabled": rule.Enabled,
 		})
 	}
-	_ = d.Set("rules", rules)
+	currentRules := []map[string]interface{}{}
+	for _, rule := range rules {
+		currentRules = append(currentRules, map[string]interface{}{
+			"name":    rule.Name,
+			"enabled": rule.Enabled,
+		})
+	}
+	fmt.Printf("Current rules: %+v", currentRules)
+	fmt.Printf("New rules: %+v", newRules)
+	areRulesSame := arePolicyRulesEquivalent(currentRules, newRules)
+	if !areRulesSame {
```

Comment:
Nit: `if !arePolicyRulesEquivalent(currentRules, newRules)` to remove the extra variable